src/app/backups/views/BackupsView/BackupsView.tsx

```diff
@@ -25,6 +34,47 @@ export default function BackupsView(): JSX.Element {
   const [isDeleteModalOpen, setIsDeleteModalOpen] = useState(false);
   const [selectedDevices, setSelectedDevices] = useState<(Device | DriveFolderData)[]>([]);
   const [backupsAsFoldersPath, setBackupsAsFoldersPath] = useState<DriveFolderData[]>([]);
+  const [folderUuid, setFolderUuid] = useState<string>();
+  const [isLoading, setIsLoading] = useState(true);
+  const [currentItems, setCurrentItems] = useState<DriveItemData[]>([]);
+  const [selectedItems, setSelectedItems] = useState<DriveItemData[]>([]);
+  const [hasMoreItems, setHasMoreItems] = useState<boolean>(true);
+  const [offset, setOffset] = useState<number>(0);
```

Comment:
It's ok this, what we are going to try to extract the maximum logic out of the view, to the handling of the pagination you can handle it with hooks, this way we abstract the state and it will be easier to maintain, since I think it's ok not to put it in redux as it's not data that needs to be accessed globally in the project.